src/main.js

```diff
@@ -121,6 +125,14 @@ function removePoster() {
    showSavedPosters();
 }
 
+// extensions
+
+
+function verifyForm(url, title, quote) {
+  if (url === "" || title === '' || quote === '') {
+    return false;
+  }
+}
```

Comment:
I think this looks really clean. Great idea to handle this in a helper function.